squarelet/organizations/serializers.py

```diff
@@ -17,7 +19,7 @@
 )
 
 
-class OrganizationSerializer(serializers.ModelSerializer):
+class OrganizationSerializer(FlexFieldsModelSerializer):
```

Comment:
Every serializer does not need to inherit from `FlexFieldsModelSerializer` - only the ones where you need the functionality, which should just be `PressPassEntitlmentSerializer` for now I believe

Reply:
Got it - I switched all the serializers back to inheriting from `serializers.ModelSerializer` except the two that needed expandable fields. (PP Entitlement and PP Invitation)